src/py/covidr/__init__.py

```diff
@@ -0,0 +1,9 @@
+from flask import Flask
+
+# EB looks for an 'application' callable by default.
+application = Flask(__name__)
+
+
+@application.route("/")
+def root():
+    return "Hello world!"
```

Comment:
it's a baby 👶

Reply:
![Unknown](https://user-images.githubusercontent.com/280924/104379006-ac4a7b00-54dd-11eb-87ed-2b8c926b02bd.jpeg)